### ISSUE TITLE
3s delays

### ISSUE THREAD

**Issue by h31p** [NicoNex/echotron#45]
after upgrading to v3.31.2, in absence of any changes in tg code - every bot update is now being processed with a 3 seconds delay.
guess, it's because of commit 6636d2e, where request limits were introduced. the default ones seems to be too agressive, and i can't find the way to change them from userland, since web client itself (and thus even public methods of it)  is unexported.

of course, reverting to v3.31.1 eliminates these 3s delays.

**Comment by NicoNex** [comment]
The update that will allow you to change this behaviour is on the way.

**Comment by NicoNex** [comment]
I've just released the version [v3.31.3](https://github.com/NicoNex/echotron/releases/tag/v3.31.3) that adds a way for the users to specify the rate limit.

```go
// SetGlobalRequestLimit sets the global rate limit for requests to the Telegram API.
// A duration of 0 disables the rate limiter, allowing unlimited requests.
func SetGlobalRequestLimit(d time.Duration)

// SetChatRequestLimit sets the per-chat rate limit for requests to the Telegram API.
// A duration of 0 disables the rate limiter, allowing unlimited requests.
func SetChatRequestLimit(d time.Duration)
```
